### PR TITLE
User guide doc

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #989

### CODE REVIEW

File: docs/src/user/guide/register-login-logout.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+- Password: Create a secure password for your account. Make sure it meets any specified password requirements.
+
+- Password Confirmation: Re-enter the password to confirm accuracy.
+
+- Country: Select the country using the dropdown menu.
+
+After filling in all the required fields, double-check to ensure accuracy. Click on 2️⃣ the `Register` button to proceed.
+
+On successful submition you will see the registration in progress message. To complete the regstration process you need to activate your account.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/register-login-logout.md]:
```suggestion
On successful submission you will see the registration in progress message. To complete the registration process you need to activate your account.
```

File: docs/src/user/guide/register-login-logout.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+![registration in progress](./img/register-login-logout-3.png)
+
+## How do I Activate account
+
+Go to your email and check the activation email for the account activation sended by the miniSASS.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/register-login-logout.md]:
```suggestion
Go to your email and check the activation email for the account activation sent by the miniSASS.
```

File: docs/src/user/guide/register-login-logout.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    
+## How do I login?
+
+Welcome to miniSASS! This user guide will walk you through the steps to log in to your miniSASS account.
+
+Click on the 1️⃣ login button to access the login page.
+
+![Login](./img/register-login-logout-5.png)
+
+On the login page fill the email address and the password in the 1️⃣ corosponding fields. Once you have entered your email address and password, click on the 2️⃣ `Login` button to proceed.
```
Comment (Jeremy-Prior) [inline docs/src/user/guide/register-login-logout.md]:
```suggestion
On the login page fill the email address and the password in the 1️⃣ corresponding fields. Once you have entered your email address and password, click on the 2️⃣ `Login` button to proceed.
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by Jeremy-Prior** [comment]
@ketankartoza please address the merge conflicts when possible

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 54 sec

**Pull request merged**